content/contribution_guidelines.md

```diff
@@ -0,0 +1,253 @@
+
+As in many open science projects, you can contribute in many ways. You can write community documents and guidelines, test the code, collect case studies, provide feedback, address issues, or start coding small bits and pieces. All of these contributions are equally important. Therefore, please do not hesitate to offer your contributions! Open science tools can only grow with community support,  so any single intention of help is extremely appreciated.
+ 
+## Table of contents
+Been here before? Already know what you're looking for? Jump to the following sections:
+- _[Joining the Neuroimaging Cookbook Project](#joining_neuroimaging_cookbook_project)_
+- _[Code of Conduct](#code_of_conduct)_
+- _[Reach Us](#reach_us)_
+- _[Contributing through GitHub](#contributing_through_github)_
+- _[Where to start: wiki, code, and templates](#where_to_start)_
+- _[Make a change with a pull request](#make_changes_pull_request)_
+    - _Example pull request_
+- _[Recognizing contributions](#recognizing_contributions)_
+
+ 
+## [Joining the Neuroimaging Cookbook Project](#joining_neuroimaging_cookbook_project)
+[Neuroimaging Cookbook Project](https://github.com/neuroimaging-cookbook/neuroimaging-cookbook.github.io) is a community project, aims to centralize code snippets from different software languages to help the community with runing/initializing their neuroimaging analysis and provide them an easier and fluid experiences in adopting open science practices in coding that serves to the same analysis purposes.
+
+The best way to start contributing to an open science community and project is to actively engage with the community resources and platforms and contributing to the ongoing projects as much as you can contribute. To start your contributions to Neuroimaging Cookbook, you might begin with reviewing and working on the currently listed [issues](https://github.com/neuroimaging-cookbook/neuroimaging-cookbook.github.io/issues)in the Neuroimaging Cookbook repository and get involved in discussions run on the [Neuroimagign Cookbook communication channels](https://mattermost.brainhack.org/brainhack/channels/neuroimaging-cookbook). Before you start,   please make sure that you read our [Code of Conduct]( ) which aims to assure creating equal, safe, and fair opportunities for every single contributor to the project.
```

Comment:
Will need to add code of conduct link. I think we would just need the relative path: "code_of_conduct"